exo-go/features/run/run.feature

```diff
@@ -0,0 +1,66 @@
+Feature: running Exosphere applications
+
+  As an Exosphere developer
+  I want to have an easy way to run an application on my developer machine
+  So that I can test my app locally.
+
+  Rules:
+  - run "exo run" in the directory of your application to run it
+  - this command boots up all the services of the application
+
+
+  Scenario: booting a functioning Exosphere application
+    Given a running "simple" application
```

Comment:
We need to describe how running an app works here, since this is the happy path spec for this feature.

```cucumber
Given I am in the root directory of the "running" example application
When starting "exo run"
Then I see "application is running"
And my machine is running the services:
  | NAME  |
  | web   |
  | users |